node-forge and postcss Vulnerabilities with Vue 2

-   Laravel Mix Version: 6.0.39
-   Node Version (`node -v`): 16.6.1
-   NPM Version (`npm -v`): 8.3.0
-   OS: Windows 11

### Description:
I use VueJS 2 for my frontend and I am using laravel-mix to compile the assets. Recently, I came across a vulnerability reported by NPM regarding node-forge which is a dependency installed due to laravel-mix and one for postcss which is caused due to vue-loader < 17.0.0 . I tried upgrading to vue-loader 17.0.0 but that broke my webpack compilation. I request your help in resolving these vulnerabilities without breaking the code.

package.json
```json
"devDependencies": {
    "axios": "^0.24.0",
    "css-loader": "^6.5.1",
    "laravel-mix": "^6.0.39",
    "lodash": "^4.17.21",
    "sass": "^1.45.1",
    "sass-loader": "^12.4.0",
    "vue": "^2.6.14",
    "vue-loader": "^15.9.8",
    "vue-template-compiler": "^2.6.14"
},
```

vulnerabilities reported in `npm audit`:
```
# npm audit report

node-forge  <1.0.0
Prototype Pollution in node-forge debug API. - https://github.com/advisories/GHSA-5rrq-pxf6-6jx5
fix available via `npm audit fix --force`
Will install laravel-mix@1.1.1, which is a breaking change
node_modules/node-forge
  selfsigned  >=1.1.1
  Depends on vulnerable versions of node-forge
  node_modules/selfsigned
    webpack-dev-server  >=2.5.0
    Depends on vulnerable versions of selfsigned
    node_modules/webpack-dev-server
      laravel-mix  >=1.2.0
      Depends on vulnerable versions of webpack-dev-server
      node_modules/laravel-mix

postcss  <8.2.13
Severity: moderate
Regular Expression Denial of Service in postcss - https://github.com/advisories/GHSA-566m-qj78-rww5
fix available via `npm audit fix --force`
Will install vue-loader@17.0.0, which is a breaking change
node_modules/@vue/component-compiler-utils/node_modules/postcss
  @vue/component-compiler-utils  *
  Depends on vulnerable versions of postcss
  node_modules/@vue/component-compiler-utils
    vue-loader  15.0.0-beta.1 - 15.9.8
    Depends on vulnerable versions of @vue/component-compiler-utils
    node_modules/vue-loader
```

Error if I try to use vue-loader v17.0.0:
```
[webpack-cli] Error: Cannot find module 'vue/compiler-sfc'
```

--- a/package.json
+++ b/package.json
@@ -88,7 +88,7 @@
         "vue-style-loader": "^4.1.3",
         "webpack": "^5.60.0",
         "webpack-cli": "^4.9.1",
-        "webpack-dev-server": "4.4.0",
+        "webpack-dev-server": "4.7.3",
         "webpack-merge": "^5.8.0",
         "webpack-notifier": "^1.14.1",
         "webpackbar": "^5.0.0-3",

